src/main/java/cn/nukkit/level/ExplosionSize.java

```diff
@@ -0,0 +1,5 @@
+package cn.nukkit.level;
+
+public class ExplosionSize {
+    public static final double DEFAULT = 4.0d;
+}
```

Comment:
I think it's better to keep de explosion sizes defined near to where it's used, there are no global default explosion size. `4.0` is the default for TNT but other explosives, like end crystal, charged creepers, etc have other default values and they aren't constant because they may change in Minecraft updates. My suggestion is to just put the `4.0` hard codded into the `simpleExplosion(Location location, Entity who)` method.